src/patternfly/components/Badge/styles.scss

```diff
@@ -0,0 +1,46 @@
+@import "../../patternfly-utilities";
+
+:root {
+  --pf-c-badge--BorderRadius: var(--pf-global--BorderRadius);
+  --pf-c-badge--FontSize: var(--pf-global--FontSize--xs);
+  --pf-c-badge--FontWeight: var(--pf-global--FontWeight--bold);
+  --pf-c-badge--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-badge--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-badge--m-read--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
+  --pf-c-badge--m-read--Color: var(--pf-global--dark--Color);
+  --pf-c-badge--m-read-alt--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+  --pf-c-badge--m-read-alt--Color: var(--pf-global--Color--dark-200);
+  --pf-c-badge--m-unread--BackgroundColor: var(--pf-global--primary-color--dark);
+  --pf-c-badge--m-unread--Color: var(--pf-global--Color--light-100);
+  --pf-c-badge--m-unread-alt--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+  --pf-c-badge--m-unread-alt--Color: var(--pf-global--primary-color);
+}
+
+.pf-c-badge {
+  display: inline-block;
+  padding-right: var(--pf-c-badge--PaddingRight);
+  padding-left: var(--pf-c-badge--PaddingLeft);
+  font-size: var(--pf-c-badge--FontSize);
+  font-weight: var(--pf-c-badge--FontWeight);
+  border-radius: var(--pf-c-badge--BorderRadius);
+
+  &.pf-m-read {
+    color: var(--pf-c-badge--m-read--Color, #292e34);
```

Comment:
is there a reason why there is a fallback here? When we used variables the first time around we did use fallbacks but haven't implemented them the second time around. @andresgalante should we be using them everywhere?

Reply:
@matthewcarleton ah darn, that's just a typo. Fallbacks are still in the component generator, and I missed removing that one. This is what the component generator creates.

```
.pf-c-component {
  color: var(--pf-c-component--Color, #292e34);
```

Reply:
👍 @andresgalante are we going to be including fallbacks?

Reply:
@andresgalante I'll remove the vars as part of this issue https://github.com/patternfly/patternfly-next/issues/271